style.css

```diff
@@ -0,0 +1,60 @@
+
+:root{
+  --font-color:rgb(80, 162, 238);
+}
+
+
+/* Styling for the loader */
```

Comment:
![image](https://user-images.githubusercontent.com/30936607/97465928-93174100-1968-11eb-99f1-c324305d75fb.png)
Hii @HarshdipD can you see currenty you are using bootstrap version 4.0.0 and the link you provided i.e bootstrap v 4.4
https://getbootstrap.com/docs/4.4/components/spinners/. Can you please check that or update the version of bootstrap so it will work fine.

Reply:
@utkarsh4321  4.4 and 4 are the same. If they're not, you can easily find docs of version 4 by googling and I'm pretty sure it will have a loader in it.

Reply:
I m already using it in popup.html it's not working.Can you please check

Reply:
You're right, it looks like v4.0 doesn't include spinners, I didn't know about this! One thing you can do is change the link to the 4.4 version and then use it. One link is in `popup.html`, and you also need to replace the link in `manifest.json` in `content_security_policy` line.
Hopefully, this does not break any styling, and no other change is required.

Reply:
I think you can skip the `manifest.json` file change because it only contains the parent domain

Reply:
ok thanks